packer/scripts/macos/macos-agentsetup.sh

```diff
@@ -1,14 +1,34 @@
 #!/usr/bin/env bash
+
+## Setup jenkins workspace
 sudo mkdir -p /var/jenkins/
 sudo chown -R ec2-user:staff /var/jenkins
+
+## Setup brew Defaults
 /usr/local/bin/brew update --preinstall
-/usr/local/bin/brew update
 /usr/local/bin/brew upgrade
-/usr/local/bin/brew install openjdk@8
-sudo ln -sfn /usr/local/opt/openjdk@8/libexec/openjdk.jdk /Library/Java/JavaVirtualMachines/openjdk-8.jdk
-/usr/local/bin/brew install python@3.7
-sudo cp /usr/local/opt/python@3.7/bin/pip3 /usr/local/bin/pip
-/usr/local/opt/python@3.7/bin/python3.7 -m pip install pipenv
-/usr/local/opt/python@3.7/bin/python3.7 -m pip install awscli
 /usr/local/bin/brew install curl 
+/usr/local/bin/brew install coreutils
+/usr/local/bin/brew install gnu-sed
```

Comment:
Do you want to  alias `gsed` to `sed` so that we don't have to duplicate scripts all over the place?

Reply:
Hey @dblock alias does/did not work with jenkins interactive shell, also symlinks for default mac packages (specific to mac ec2 instances) throws read only file system error's, hence I have added to the `PATH` using `bashrc` to make sure jenkins interactive shell uses this `PATH`.

Reply:
Correction, alias does not work with ANY non-interactive shell.